src/modules/site-settings/edit-site-details.tsx

```diff
@@ -278,6 +282,18 @@ export default function EditSiteDetails( { currentWpVersion, onSave }: EditSiteD
 										</div>
 									</div>
 								) }
+
+								{ useCustomDomain && (
+									<div className="flex items-center gap-2 mt-4">
+										<input
+											type="checkbox"
+											id="enable-ssl"
+											checked={ enableSSL }
+											onChange={ ( e ) => setEnableSSL( e.target.checked ) }
+										/>
+										<label htmlFor="enable-ssl">{ __( 'Enable SSL' ) }</label>
```

Comment:
```suggestion
										<label htmlFor="enable-ssl">{ __( 'Enable HTTPS' ) }</label>
```
Per my other suggestion of updating the label